Lombiq.HelpfulExtensions/Assets/Scripts/target-blank.js

```diff
@@ -5,16 +5,11 @@
 
         for (let i = 0; i < links.length; i++) {
             if (!links[i].href.match(/^mailto:/) &&
-                (links[i].hostname !== currentHostname &&
-                (!links[i].href.match(/^javascript:/i)))) {
+                links[i].hostname !== currentHostname &&
+                !links[i].href.match(/^javascript:/i)) {
                 links[i].setAttribute('target', '_blank');
             }
         }
     }
-    window.addEventListener(
-        'load',
-        () => {
-            window.setTimeout(targetBlank, 100);
-        },
-        false);
+    document.addEventListener('DOMContentLoaded', targetBlank);
```

Comment:
Let's also rename the function to something more explicit like `addTargetBlank`.